src/web/docusaurus/docs/tools-and-technologies/supabase.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 10
+---
+
+# Supabase
+
+## What is [Supabase](https://supabase.com/docs/)
+
+[Supabase](https://supabase.com/docs/) is an application development software that includes many backend services:
+
+- Database: A dedicated scalable Postgres database
+- Studio: A web dashboard that includes Table & SQL editors, Database management, and API documentation.
+- Rest API: Auto-generated API for your database
+- Realtime: Realtime subscription to database changes
+- Authorization: User management with Row Revel Security
+
+## Architecture
+
+![Supabase Architecture](../../static/img/supabase-architecture.png)
+
+## Self-hosting
+
+Unlike the [hosted platform](https://app.supabase.io/), Telescope self-hosts and manages the infrastructure where the containerized Supabase runs.
+
+Supabase consists of many containerized images defined in [docker/supabase](https://github.com/Seneca-CDOT/telescope/tree/master/docker/supabase)
+
+### Configuration
+
+There are 4 important keys that help us secure our database.
+
+- `POSTGRES_PASSWORD`: POSTGRES database password
+- `ANON_KEY`: A public API key, used for interacting with the database through REST interface but restricted by RLS
+- `SERVICE_ROLE_KEY`: A private admin key used on the server-side and is not restricted by RLS
+- `JWT_SECRET`: JWT secret used for verifying the signature of backend-issued JWT token
+
+### Role level security - RLS
+
+Row-level security (RLS for short) is a PostgreSQL security feature that allows database administrators to define policies to control how specific rows of data display and operate for one or more roles.
```

Comment:
I would change "display and operate" to "...how specific rows of data are accessed and modified based on user roles."